cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -185,9 +184,9 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
 
 
     PublishXBlockModal = CourseOutlineXBlockModal.extend({
-        events : {
+        events : _.extend({}, CourseOutlineXBlockModal.prototype.events, {
             'click .action-publish': 'save'
-        },
+        }),
```

Comment:
@ehteshamkafeel Why we need to extend the events property of `SettingsXBlockModal` and `PublishXBlockModal`